### PR TITLE
Add image backgrounds, animations, and small search bugfixes

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Hodgepodge PR of a few things. 

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Physical device

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-image-animation

File: packages/mobile/src/components/now-playing-drawer/NowPlayingDrawer.tsx [hunk 1]
```diff
@@ -165,6 +168,16 @@ const NowPlayingDrawer = (props: NowPlayingDrawerProps) => {
     [drawerPercentOpen]
   )
 
+  const onPanResponderRelease = useCallback(
```
Comment (dylanjeffers) [inline packages/mobile/src/components/now-playing-drawer/NowPlayingDrawer.tsx:171]: nice

File: packages/mobile/src/hooks/usePressScaleAnimation.ts [hunk 2]
```diff
@@ -9,19 +9,21 @@ export const usePressScaleAnimation = (
   const scale = useRef(new Animated.Value(1)).current
 
   const startPress = useCallback(() => {
-    Animated.timing(scale, {
+    Animated.spring(scale, {
```
Comment (dylanjeffers) [inline packages/mobile/src/hooks/usePressScaleAnimation.ts:12]: cool

File: packages/mobile/src/screens/search-screen/SearchBar.tsx [hunk 3]
```diff
@@ -43,6 +44,14 @@ export const SearchBar = () => {
     [dispatch, fetchSearchDebounced, setClearable]
   )
 
+  useEffect(() => {
+    if (query !== searchResultQuery && searchResultQuery !== '') {
+      console.log('neq')
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/search-screen/SearchBar.tsx:49]: log

File: packages/mobile/src/screens/search-screen/SearchScreen.tsx [hunk 4]
```diff
@@ -57,21 +57,30 @@ export const SearchScreen = () => {
...
+      searchResultQuery &&
+      searchQuery === searchResultQuery &&
+      !hasResults
+    ) {
       return <EmptySearch query={searchResultQuery} />
     }
     return <SearchHistory />
   }
 
+  const renderSearchBar = () => {
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/search-screen/SearchScreen.tsx:71]: nit but can just be `searchBar` not a fn

File: packages/web/src/common/store/search-bar/sagas.ts [hunk 5]
```diff
@@ -82,9 +82,6 @@ function* watchSearch() {
     if (lastTask) {
       // cancel is no-op if the task has already terminated
       yield* cancel(lastTask)
-
-      // Reset the search bar state
-      yield* put(searchActions.clearSearch())
```
Comment (dylanjeffers) [inline packages/web/src/common/store/search-bar/sagas.ts:87]: interesting, what was happening here?

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-image-animation

**Review by dylanjeffers — APPROVED** [review]
nice changes

File: packages/web/src/common/store/search-bar/sagas.ts (hunk 5 above)
Comment (raymondjacobson) [reply packages/web/src/common/store/search-bar/sagas.ts:87]: UX is way better if we don't clear the autocomplete each time you type a char which leads to the history view showing. Same with web (tested there too)

File: packages/mobile/src/screens/search-screen/SearchScreen.tsx (hunk 4 above)
Comment (raymondjacobson) [reply packages/mobile/src/screens/search-screen/SearchScreen.tsx:71]: was just following the pattern. let's chat at next client syncobinko

**Pull request merged**